libr/util/file.c

```diff
@@ -228,13 +229,19 @@ R_API char *r_file_path(const char *bin) {
 		return NULL;
 	}
 	path_env = (char *)r_sys_getenv ("PATH");
+	bin_fmt = R_JOIN_2_PATHS ("%s", "%s");
```

Comment:
using a var to hold a format string for just appending .exe hurts a bit, you can just have a const chr *extension = ""; to be set on WINDOWS and then use that variable in the format string below. you will remove like 5 lines of code and simplify the logic quite a bit